projects/graph/src/graph.py

```diff
@@ -76,18 +76,18 @@ def dft(self, starting_node):
         s = Stack()
         visited = set()
         #push the starting node
-        s.push()
+        s.push(starting_node)
         #while the node is not empty
         while s:
             #pop a node from the stack
-            popped = s.pop(0)
+            popped = s.pop()
```

Comment:
Hoping you understand the concepts now Ian, please let me know if you have any troubles